build1.sh

```diff
@@ -2,13 +2,14 @@
 #packages required to edit
 sudo apt-get install -qq squashfs-tools genisoimage
 #downloading the ISO to edit
-wget -q https://github.com/fossasia/meilix/releases/download/untagged-c9f68d2fa6b3d1cdbd99/meilix-zesty-20170611-i386.iso
+wget -q https://github.com/fossasia/meilix/releases/download/untagged-15efc7e1086f3686d999/meilix-zesty-20170718-i386.iso
```

Comment:
This is the latest one, but it should get varied automatically as other latest release come.